### PR TITLE
Set up CI testing via Github actions

### PR DESCRIPTION
**Pull request by rschroll**
This adds an action to run the Django tests on every push.  To make this easier to set up, I switched the default database to use SQLite.  Depending on what we do in #10, this may or may not be the right way to handle things.

Right now, tests are run on Python 3.7, only because that's what I've been using.  Once we pick a preferred version (see #9), we can adjust that here.  Alternatively, Github actions make it relatively easy to run the same tests against multiple Python versions.

### CODE REVIEW

**Comment by rschroll** [comment]
The action ran on my account, but it didn't link the result to this PR, for whatever reason.  You can see the output here: https://github.com/rschroll/GO3/runs/422149229.

File: var/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*
```
Comment (aaronoppenheimer) [inline var/.gitignore]: Doesn't this just silence git from reporting anything that isn't added?

Comment (aaronoppenheimer) [reply var/.gitignore]: huh - actually, it doesn't look like anything is different.

**Comment by aaronoppenheimer** [comment]
That is cool - it's almost like real devops! Thanks for setting it up.

I'm fine with using sqlite by default, and I think using a local settings override file ('from settings_local import *') is best just because it's simplest.

**Comment by aaronoppenheimer** [comment]
I've pushed a change to settings.py to import settings_local.py, if it exists. works like a champ.

I'll now stop developing directly into the master branch. Right now it's set up so that I can merge without requiring a review - if I do that, do you get notified?

A

**Comment by rschroll** [comment]
I haven't gotten notifications so far, but that's probably because I just joined the project.  I'm sure there's a notification setting where I can tweak it to my heart's content.  You keep up whatever workflow works for you; I'll let you know if it creates any problems on my end.

Re the .gitignore in var: A trick I've used elsewhere is to use a var folder for all sorts of files that vary from developer to developer.  That's where I've stuck the default DB, for example.  By putting the .gitignore file in the directory, rather than adding an entry to the main copy, it ensure that this directory gets created on cloning the repo.  If we were to allow multiple settings files, sticking them in var would make sense.  But the current solution doesn't admit that, so I think the current location makes sense.

**Comment by aaronoppenheimer** [comment]
Ah, I see - I didn't notice var was added. Makes sense.

**Pull request merged**